src/types/share.ts

```diff
@@ -21,4 +23,9 @@ export interface PublishRequest {
   name: string;
   path: string;
   version: string;
+  fileNameMapping: Map<string, string>;
+}
+
+export interface PublishAttachmentInfo extends Omit<DialFile, 'contentLength'> {
+  path: string;
```

Comment:
Why do you need one more path? Are relative path or absolute path not applicable for you?